test/cmd/newapp.sh

```diff
@@ -107,6 +107,9 @@ os::cmd::try_until_success 'oc get imagestreamtags myruby:latest'
 os::cmd::expect_success 'oc new-app myruby~https://github.com/openshift/ruby-hello-world.git --dry-run'
 os::cmd::expect_success 'oc delete is myruby'
 
+# Ensure clear error message wrt templates container CRDs and new-app appears
+os::cmd::expect_failure_and_text 'oc new-app -f test/testdata/newapp/templates-with-crd.yaml' 'error: The template contained an object type unknown to '
```

Comment:
@gabemontero file name is template-with-crd.yaml

May also want to run `hack/update-generated-bindata.sh`

Reply:
yep caught that @adambkaplan but thanks 

also, I ran generated bindata previously and it produced nothing new ... only pertains to test/extended artifacts, not test/cmd